### PR TITLE
tweak logout url

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: src/unicef_security/views.py [hunk 1]
```diff
@@ -15,5 +15,9 @@ def get_context_data(self, **kwargs):
 
 class UNICEFLogoutView(RedirectView):
     def get_redirect_url(self, *args, **kwargs):
-        return f'https://{settings.TENANT_B2C_URL}/{settings.TENANT_ID}/{settings.POLICY}/oauth2/v2.0/' \
+
+        base_url = getattr(settings, 'TENANT_B2C_URL', 'login.microsoftonline.com')
```
Comment (robertavram) [inline src/unicef_security/views.py:19]: if these are needed for default values please add them to config file same as UNICEF_EMAIL. you can import django conf.settings there as well.

File: src/unicef_security/views.py [hunk 2]
```diff
@@ -15,5 +15,9 @@ def get_context_data(self, **kwargs):
...
+        tenant_id = getattr(settings, 'SOCIAL_AUTH_TENANT_ID', settings.TENANT_ID)
+
+        return f'https://{base_url}/{tenant_id}/oauth2/v2.0/' \
```
Comment (robertavram) [inline src/unicef_security/views.py:22]: is the policy no longer required?

Comment (robertavram) [inline src/unicef_security/views.py:22]: is the policy no longer required? /  will this not break things in other places?

**Review by robertavram — CHANGES_REQUESTED** [review]

File: src/unicef_security/views.py (hunk 2 above)
Comment (domdinicola) [reply src/unicef_security/views.py:22]: looks is not

File: src/unicef_security/views.py (hunk 1 above)
Comment (robertavram) [reply src/unicef_security/views.py:19]: not sure that you read my message correctly.

File: src/unicef_security/views.py (hunk 2 above)
Comment (robertavram) [reply src/unicef_security/views.py:22]: if it's all the same, I'd like to keep policy in. I'm not super-familiar with the inner workings of AD logout, but in my experience, being more specific is usually a good thing. (From the AD docs is where I've picked up that policy that is templated in the URL) - Does it break functionality in your app if we keep it in?

**Pull request merged**